scripts/gogetcookie.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
```

Comment:
the script is unused

+ it looks invalid 
https://github.com/hashicorp/terraform/commit/336b352d6fe4de90392691d892557d5fa2cd07d4

i'm not sure we really need it, maybe we need some research about it.

Reply:
Removed it